server/data/services/mathsoc-office.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
can you add this file to the data validation pipeline? otherwise, we won't be able to edit it through the cms.